js/replay-embed.js

```diff
@@ -7,8 +7,8 @@ document.write('<script src="https://play.pokemonshowdown.com/js/lib/html-saniti
 document.write('<script src="https://play.pokemonshowdown.com/js/lib/soundmanager2-nodebug-jsmin.js"></script>');
 document.write('<script src="https://play.pokemonshowdown.com/js/config.js?a6"></script>');
 document.write('<script src="https://play.pokemonshowdown.com/js/battledata.js?a6"></script>');
-document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini.js?a6"></script>');
-document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini-bw.js?a6"></script>');
+document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini.min.js?a6"></script>');
+document.write('<script src="https://play.pokemonshowdown.com/data/pokedex-mini-bw.min.js?a6"></script>');
```

Comment:
> `.min.js`

Honestly, if we're doing this, we should just combine them into one file. They're not big enough to be worth the on-demand loading.